index.html

```diff
@@ -6,6 +6,7 @@
     <title>Nep huiswerkopdracht</title>
 </head>
 <body>
-    <h2>Dit is een andere titel, die gaat niet over Pim</h2>
```

Comment:
Super origineel! I like it. Maar niet echt een titel voor een website. Misschien kun je dat nog even aanpassen